server/routes/github.coffee

```diff
@@ -28,26 +28,38 @@ module.exports.setup = (app) ->
           return log.error err if err?
           githubUser = JSON.parse response
           log.info 'Got GitHub auth callback response', githubUser
-          emailLower = githubUser.email?.toLowerCase()
-          if not emailLower
-            return errors.serverError res, "Problem finding GitHub user with that identity."
+          
+          request.get {uri: 'https://api.github.com/user/emails', headers: headers}, (err, r, response) ->
+            return log.error err if err?
+            githubUserEmails = JSON.parse response
+            log.info 'Got GitHub user emails', githubUserEmails
+          
+            for email in githubUserEmails
+              if email.primary == true
```

Comment:
Use `is` instead of `==` in CoffeeScript. Alternatively, use [_.find](https://lodash.com/docs#find).